terraform/environments/bootstrap/delegate-access/policies.tf

```diff
@@ -96,6 +96,7 @@ data "aws_iam_policy_document" "developer_additional" {
   #checkov:skip=CKV_AWS_109
   #checkov:skip=CKV_AWS_111
   #checkov:skip=CKV_AWS_110
+  #checkov:skip=CKV_AWS_356
```

Comment:
can we have a reasoning why we are suppressing this message, please? like in the previous file:
`# checkov:skip=CKV_AWS_356: "Cannot restrict by KMS alias so leaving open"`